packages/cardpay-reward-programs/cardpay_reward_programs/main.py

```diff
@@ -13,34 +14,19 @@
 load_dotenv()
 
 cached_client = S3Client(
-    local_cache_dir="mycache",
+    local_cache_dir=".cache",
     boto3_session=Session(
-        aws_access_key_id=os.getenv("AWS_ACCESS_KEY_ID"),
-        aws_secret_access_key=os.getenv("AWS_SECRET_ACCESS_KEY"),
     ),
 )
 cached_client.set_as_default_client()
 
 
-def select_rule(name, core_parameters, user_defined_parameters):
-    rule_constructor = globals()[name]
```

Comment:
Instantiating a class based on allowing user input to select the code to run feels risky.